spec/models/miq_server_spec.rb

```diff
@@ -411,4 +411,11 @@
       expect(described_class.new(:status => "stopped").active?).to be_falsey
     end
   end
+
+  describe ".raise_failover_executed_event" do
+    it "raises ''db_failover_executed'' event" do
+      expect(MiqEvent).to receive(:raise_evm_event).with(MiqServer.my_server, 'db_failover_executed')
```

Comment:
yes, it looks weak. It just to confirming that ```MiqEvent.raise_evm_event``` was called.
```raise_evm_event```  has its own coverage and i was thinking that it should be sufficient. (Also tested it end-to end and saw in log that event was dispatched to alerts and policies). 
Will think of better way to test it ...